LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Wasn't sure what to pick here, I think MIT is generally good since it extends the reach of the project by allowing commercial use, but it's definitely not for everyone.